external-task-worker/index.js

```diff
@@ -0,0 +1,58 @@
+const {HttpClient} = require('@essential-projects/http');
+const {
+  ExternalTaskApiClientService,
+  ExternalTaskApiExternalAccessor,
+  ExternalTaskWorker,
+} = require('@process-engine/external_task_api_client');
```

Comment:
Benutz besser `@process-engine/consumer_api_client`. Da befindet sich auch ein ExternalTaskWorker drin.
`@process-engine/external_task_api_client` ist eigentlich nur noch verfügbar, damit z.B. Team X nicht gezwungen ist ihren Code umzuschreiben, es soll aber eigentlich auf lange Sicht abgelöst werden.

```suggestion
} = require('@process-engine/consumer_api_client');
```